jekyll/index.html

```diff
@@ -1,40 +1,22 @@
 ---
 layout: classic-docs
-collection: cci1
+page-type: homepage
+title: CircleCI Documentation
 ---
 
-{% for category in site.data.categories %}
-{% if category.section == "cci1" %}
-
-{% if category.index %}
-	{% assign catFile = category.index %}
-{% else %}
-	{% assign catFile = category.slug %}
-{% endif %}
-
 <div class="category-section">
-	<img src="{{ site.baseurl }}/assets/img/icons/{{ category.icon }}" class="logo" alt="Category Icon" />
-	<h2>{{ category.name }}</h2>
-	<ul class="list-unstyled">
-	{% assign docs_found = 0 %}
-	{% for doc in site.cci1 %}
-		{% if doc.categories contains category.slug and doc.slug != catFile and doc.hide != True and doc.section == "cci1" %}
-			{% assign docs_found = docs_found | plus: 1 %}
-			{% if docs_found < 4 %}
-				{% if doc.short-title %}
-					<li class="{% if page.path contains doc.url %}active{% endif %}"><a href="{{ site.baseurl }}{{ doc.url }}">{{ doc.short-title }}</a></li>
-				{% else %}
-					<li><a href="{{ site.baseurl }}{{ doc.url }}">{{ doc.title }}</a></li>
-				{% endif %}
-			{% endif %}
-		{% endif %}
-	{% endfor %}
-
-	{% if docs_found > 3 %}
-		{% assign more = docs_found | minus: 3 %}
-		<li><em><a href="{{ site.baseurl }}/{{ catFile }}/">{{ more }} more</a></em></li>
-	{% endif %}
-	</ul>
+	<h2><a href="/docs/1.0/">1.0 Docs</a></h2>
+		<p>Full documentation for CircleCI. Guides, troubleshooting and API reference.</p>
+</div>
+<div class="category-section">
+	<h2><a href="/docs/enterprise/">Enterprise Docs</a></h2>
+		<p>Documentation specific to customers administering self-hosted, behind the firewall CircleCI Enterprise.</p>
+</div>
+<div class="category-section">
+	<h2><a href="https://discuss.circleci.com/">Community</a></h2>
+		<p>We welcome your feedback, questions and feature requests on our community forums - Discuss. Searching the forums is a great way to find solutions to specific issues not covered byt the docs.</p>
+</div>
```

Comment:
This page needs it's own layout file. Again, can be done after this PR. The empty sidebar looks bad and probably shouldn't exists at all for this page.

The only thing I don't really like about this page is that the sections are manual. We can probably move the descriptions to where the collections are defined and loop through them.

I REALLY like having Discuss and Support mentioned. Great touch.